pkg/apis/sources/v1beta1/apiserver_conversion_test.go

```diff
@@ -204,7 +204,7 @@ func TestApiServerSourceConversionRoundTripUp(t *testing.T) {
 	}
 }
 
-// This tests round tripping from a higher version -> v1alpha1 and back to the higher version.
+// This tests round tripping from a higher version -> v1beta1 and back to the higher version.
```

Comment:
this still tests v1beta1 -> v1alpha1

Reply:
unless something was deleted somewhere else?

Reply:
I took a look at the code. The `test.in` is v1 version,  the `down` variable is v1beta1 version. The UT is to test thatround tripping from a v1 -> v1beta1 and v1beta1 ->v1

Reply:
I agree with @capri-xiyue, this looks like it's testing v1 -> v1beta1.